### ISSUE TITLE
Failed test "70-test_sslcbcpadding.t"

### ISSUE THREAD

**Issue by sebastianas** [openssl/openssl#4688]
I re-run the test over 7k times and it happend only once. This was on Debian-AMD64. I have the full build-log, the following is only the part of the failed test. Is this enough information to see what happend?

[70-test_sslcbcpadding.t_failed.txt](https://github.com/openssl/openssl/files/1447798/70-test_sslcbcpadding.t_failed.txt)

**Comment by mattcaswell** [comment]
This is almost certainly some kind of race condition in the TLSProxy test utility. I've looked at the log but I can't see anything that might explain it. Probably either the socket connecting the proxy to the server or the socket connecting the proxy to the client has died unexpectedly before the test was complete. The log doesn't give any hints as to why though.

**Comment by sebastianas** [comment]
Died unexpectedly as in a race condition on done/termination?
It would be nice to get some debugging from that module.
Feel free to close it if there is nothing that can be done.

**Comment by sebastianas** [comment]
On 2018-01-22 17:40:12 [+0000], mattcaswell wrote:
> Is this still an issue with the latest git version (which OpenSSL version is this)? There have been a lot of TLSProxy fixes recently.

This must have been the latest openssl 1.1.0 release at that time (g). I
wasn't able to reproduce it since. I assumed it was closed / will be
closed once TLSProxy has more debug output so that it is easier to
narrow such cases down.

Sebastian

**Comment by mattcaswell** [comment]
Assumed fixed by the latest TLSProxy changes - closing. If this happens again please open a new issue.
